custom.css

```diff
@@ -0,0 +1,181 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+html {
+  background-color: #dfe1e6;
+}
+
+.container {
+  margin: 114px 24px 0 24px;
+  width: 100%;
+  display: grid;
+  gap: 88px;
+}
+
+.card-body {
+  border-radius: 16px;
+  padding: 16px;
+  background-color: #fff;
+  height: 100%;
+  width: 327px;
+}
```

Comment:
Kindly update the works section to match the figma design better🙏

Your design:
<img width="213" alt="image" src="https://user-images.githubusercontent.com/37116322/227596252-377748b5-cd5e-40dd-b92b-3a4038efc98c.png">

Figma design:
<img width="192" alt="image" src="https://user-images.githubusercontent.com/37116322/227596326-13059daf-3912-49e5-a1cb-3ac38bac914a.png">

A few things to note:
- Please make the left and right margins of each card even
- Please update to background color of the section to match the template. I believe `#f7f7f9` should do it